miso-web/src/main/webapp/scripts/form_project.js

```diff
@@ -94,6 +101,18 @@ FormTarget.project = (function($) {
           title: 'Clinical',
           data: 'clinical',
           type: 'checkbox'
+        }, {
+          title: 'Use Secondary Naming Scheme',
+          data: 'secondaryNaming',
+          type: 'checkbox',
+          include: !!config.naming.secondary,
+          onChange: function(newValue, form) {
+            var scheme = config.naming[newValue ? 'secondary' : 'primary'];
+            form.updateField('shortName', {
+              required: scheme.shortNameRequired,
+              disabled: !scheme.shortNameModifiable
+            });
+          }
```

Comment:
Would it make sense to move this before the "Short Name" column, as that'll be affected by changes to this checkbox (and could lead to users having previously-valid data be marked invalid, or have to enter in invalid data which will then be validated by checking this box as they fill in left-to-right)?

Reply:
Yeah, makes sense. Will move

Reply:
Moved